bugbug/model.py

```diff
@@ -250,7 +250,9 @@ def print_feature_importances(
                 else:
                     shap_val.append("-" + str(importance))
 
-                top_feature_names.append(feature_names[int(index)])
+                top_feature_names.append(
+                    f"{feature_names[int(index)]} \nvalue: {X.toarray()[:,int(index)]}"
```

Comment:
Instead of:
```
FEATURE_NAME 
value: [VALUE]
```
Let's do:
```
FEATURE_NAME = VALUE
```

The other characters (newline, `value` and the parenthesis) are not really adding info but just making the output more crowded.

Reply:
I tried this but then the length of feature names become too long and it doesn't align good then

Reply:
Can you show me? The other option is FEATURE_NAME\nVALUE

Reply:
![Selection_007](https://user-images.githubusercontent.com/47722139/62426659-3e552e00-b705-11e9-8391-6854e767947e.png)

Reply:
Couldn't we just reduce the 6 at https://github.com/mozilla/bugbug/pull/847/files#diff-5602bf3d29e24bb4a2603298fae3ebbdR272?

(Remove the "[" and "]" too)